### ISSUE TITLE
Custom tasks not working properly

### ISSUE THREAD

**Issue by pmb0** [geddski/grunt-release#115]
Hi,

there is still an issue with the custom tasks feature. Example Gruntfile.js:

```js
module.exports = function(grunt) {
    grunt.initConfig({
        release: {
            options: {
                afterBump: ['hello', 'hello2']
            }
        }
    });
    grunt.loadNpmTasks('grunt-release');
    grunt.registerTask('hello', 'Says hello', function() {grunt.log.writeln('hello');});
    grunt.registerTask('hello2', 'Says hello2', function() {grunt.log.writeln('hello2');});
};
```

Output:

```sh
$ grunt release
Running "release" task
>> Release dry run.
>> running afterBump
>> bumped version of package.json to 1.0.1
>> staged package.json
>> Committed all files
>> created new git tag: 1.0.1
>> pushed to remote
>> pushed new tag 1.0.1 to remote
>> published version 1.0.1 to npm

Running "hello" task
hello

Running "hello2" task
hello2
```

`afterBump` tasks run after other stuff finishes.

**Comment by didlich** [comment]
very important for, I vote for this
+1

**Comment by drublic** [comment]
Thank you for the heads up. #118 is the same as this bug, I think. I've just merged #122 now. Should be fixed. New version coming.
